### DISCUSSION TITLE
I get same error for each service WFLYEJB0405: No Jakarta Enterprise Beans found with interface of type 'org.ejbca.core.ejb.EjbBridgeSessionLocal' and name 'EjbBridgeSessionBean' for binding java:module/env/EjbBridgeSession" ....

### DISCUSSION THREAD

**Discussion by TByte007** [Keyfactor/ejbca-ce#468] [Q&A] [answered]
I'm trying to upgrade an old installation of EJBCA but when I deploy it I get this same error for (I think) each service:
`WFLYEJB0405: No Jakarta Enterprise Beans found with interface of type 'org.ejbca.core.ejb.EjbBridgeSessionLocal' and name 'EjbBridgeSessionBean' for binding java:module/env/EjbBridgeSession"`

What am I missing in the installation process?

**Comment by primetomas** [comment]
You need to provide all information about your components.
What version of EJBCA, Java, WildFly, etc.

Sometimes WildFly get stuck with cached information an then a simple command like this usually helps.
rm -rf wildfly/standalone/tmp
rm -rf wildfly/standalone/data

**Comment by FaysalMossi** [comment]
you can also run this command "**ant deployear**" without "**-q clear**"

**Comment by TByte007** [comment, marked as answer]
> You need to provide all information about your components. What version of EJBCA, Java, WildFly, etc.
> 
> Sometimes WildFly get stuck with cached information an then a simple command like this usually helps. rm -rf wildfly/standalone/tmp rm -rf wildfly/standalone/data

I found out what the problem is. EJBCA is not supported on Wildfly higher than 26. Something to do with Jakarta's version.

**Comment by mike-agrenius-kushner** [comment]
Yeah. There is no backwards compatibility between Jakarta8 (where we are now) and Jakarta10 (which is WF27+). The problem for us is that we have users who rely on JBoss EAP, who are still stuck on WF23. The next version of EAP should have been out in Q4 23, so we can't abandon those users until Redhat gets the new EAP released.
